src/System.Composition.sln

```diff
@@ -0,0 +1,97 @@
+﻿
```

Comment:
Hmm, if the assemblies are shipped together then I'd move everything into a common src\System.Composition as to me the "folder == nuget package" convention is more important than "nuget package == single assembly".